rts/System/Sync/SyncedPrimitiveBase.h

```diff
@@ -44,7 +44,7 @@ namespace Sync {
 	static inline void Assert(const void* p, unsigned size, const char* msg) {
 		AssertDebugger(p, size, msg);
 #ifdef SYNCCHECK
-		assert(CSyncChecker::InSyncedCode());
+		// assert(CSyncChecker::InSyncedCode());
```

Comment:
should this stay so?

Reply:
No, no it shouldn't. I try to keep that change from appearing in git commits, but it isn't always easy! :D - I'll keep this comment open and make it the last thing I fix.

Reply:
Done.